ready_recipes/arm_pyart/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.6.0" %}
+
+package:
+    name: arm_pyart
+    version: {{ version }}
+
+source:
+    fn: arm_pyart-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/a/arm_pyart/arm_pyart-{{ version }}.tar.gz
+    md5: 6c382dda78193c730a6a47d2d7b18eeb
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - trmm_rsl   # [not win]
+
+    run:
+        - python
+        - numpy
```

Comment:
I though so too, but when `numpy x.x` is specified the Python 3.5 Windows builds fail during the testing step due to "Unsatisfiable package specifications.", [x86](https://ci.appveyor.com/project/pelson/staged-recipes/build/1.0.249/job/1iqv3xb1q0jkv0md) and [x64](https://ci.appveyor.com/project/pelson/staged-recipes/build/1.0.249/job/0nuua16getk2k67f).  Not sure what is causing this but I think using just `numpy` in the run dependency should be fine, conda should still enforce the version requirement that is defined in the package name.